### DISCUSSION TITLE
start milvus with docker-compose,milvus-etcd start failed（使用docker 容器启动milvus,milvus-etc组件启动失败）

### DISCUSSION THREAD

**Discussion by gzg25895381** [milvus-io/milvus#20150] [Q&A and General discussion]
环境为ubuntu，带gpu的机器，gpu为特斯拉4
下载的docker-compose.yml为官网提供的命令下载，命令如下：
wget https://github.com/milvus-io/milvus/releases/download/v2.1.4/milvus-standalone-docker-compose.yml -O docker-compose.yml
docker 版本：20.10.12
docker-compose 版本：docker-compose version 1.26.2, build eefe0d31
执行命令（start command）：sudo docker-commpose up -d
执行结果(result):使用 docker-compose ps查看服务状态，milvus-etcd启动失败
使用 sudo docker logs -t -f milvus-etcd 查看milvus-etcd的日志显示结果如下：
2022-10-28T02:20:50.906620469Z exec /usr/local/bin/etcd: exec format error
试过（tryed）：
sudo docker-compose down
sudo rm -rf volumes
sudo docker-compose up -d
问题依然存在（not works）
请问这种问题应该怎么排查，或者从哪个方向入手，谢谢（what can i do next or how to fix this,thanks）

**Comment by yhmo** [comment]
可以试一下把docker-compose.yml里除了etcd之外的service都去掉，单独起etcd看其日志是否仍是exec format error，多半是系统兼容性问题。

```
Here are four common reasons why a binary file might not work:
- They compile it for a different version of your [OS].
- They create it with another compiler.
- The file is corrupt.
- The operating system doesn’t support executables in that format.
```
According to this article:
https://www.starlanguageblog.com/cannot-execute-binary-file-exec-format-error/
